src/modules/hip/kernel/glitch.hpp

```diff
@@ -0,0 +1,278 @@
+#include <hip/hip_runtime.h>
+#include "rpp_hip_common.hpp"
+
+template <typename T>
+__device__ __forceinline__ void rpp_hip_load1_glitch(T *srcPtr, uint srcStrideH, float locSrcX, float locSrcY, int4 *roiPtrSrc_i4, float *dst, int channelStride, int chn)
+{
+    int srcIdx = locSrcY * srcStrideH + locSrcX * channelStride + chn;
```

Comment:
where is roiPtrSrc_i4 used inside this function?
srcStrideH is uint and channelStride is int
can we combine these 2 into single uint2 variable?

Reply:
Done